pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -104,6 +111,25 @@ func (r *rbac) createResources(ctx context.Context) error {
 		return nil
 	}
 
+	exist, err := checkIfInstallerSetExist(ctx, r.operatorClientSet, r.version, r.tektonConfig, componentName)
+	if err != nil {
+		return err
+	}
+	if !exist {
+		if err := createInstallerSet(ctx, r.operatorClientSet, r.tektonConfig,
+			r.version, componentName, "rbac-resources"); err != nil {
```

Comment:
why is 2 names necesary 1. the constant `componentName` and then `rbac-resources`.